ledger/ledger-api-client/src/it/scala/com/digitalasset/ledger/client/LedgerClientIT.scala

```diff
@@ -64,6 +64,15 @@ final class LedgerClientIT
       }
     }
 
+    "get api version" in {
+      for {
+        client <- LedgerClient(channel, ClientConfiguration)
+        version <- client.versionClient.getApiVersion()
+      } yield {
+        version should fullyMatch regex """1\.\d+\.\d+"""
```

Comment:
Yeah, looks good to me. My only comment is that we should probably check against SemVer rather than `1.\d.\d` here. See https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
That's a valid point! I'll modify the PR 👍